### ISSUE TITLE
Loss_fake for both D and G rapidly reach to 0.0

### ISSUE THREAD

**Issue by leocnj** [hujinsen/awesome-pytorch-StarGAN-VC#2]
When trying the code on the VCC data set, I got the following loss plots. The loss_fake for both D and G reached to 0.0 after about 3000 iterations and became flat. Flat loss curves seem not right for NN training. I am wondering whether you also observed the same loss pattern before?
![image](https://user-images.githubusercontent.com/4459248/51080979-26de3580-1720-11e9-8189-1ae568eb3b2b.png)

**Comment by Pydataman** [comment]
@leocnj have you solved it?

**Comment by hujinsen** [comment]
> @leocnj have you solved it?

fixed bugs. please checkout new version.

**Comment by hujinsen** [comment]
> When trying the code on the VCC data set, I got the following loss plots. The loss_fake for both D and G reached to 0.0 after about 3000 iterations and became flat. Flat loss curves seem not right for NN training. I am wondering whether you also observed the same loss pattern before?
> ![image](https://user-images.githubusercontent.com/4459248/51080979-26de3580-1720-11e9-8189-1ae568eb3b2b.png)

fixed bugs. please checkout new version.

**Comment by hujinsen** [comment]
fixed bugs. please checkout new version.
